.gitignore

```diff
@@ -1,6 +1,5 @@
 dist
 package
 .DS_Store
-.slack/apps.dev.json
 .slack/apps.json
 .slack/config.json
```

Comment:
question: should we remove these 2? I don't see them included in the other templates and these files should be checked into version control for the dev.

Reply:
Yeah I'm not sure why this is set up like this 🤔 I'm not sure if these samples are used somewhere else

But good point I'll remove this